docs/pages/brownfield/integration-with-existing-apps.mdx

```diff
@@ -0,0 +1,654 @@
+---
+title: Integrating a React Native view into existing native apps
+sidebar_title: Integrated approach
+description: Add Expo and React Native to existing native apps using the "integrated" approach.
+hideTOC: true
+---
+
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+import { PlatformTag } from '~/ui/components/Tag/PlatformTag';
+
+React Native offers a flexible development strategy. Whether you're launching a new app or looking to integrate an existing native app, it provides a powerful solution. You can leverage its capabilities to introduce new features or views incrementally, allowing your team to adopt modern, cross-platform development without a complete rewrite.
+
+This guide will walk you through the steps to add a React Native view into an existing native app using the "integrated" approach.
+
+## Prerequisites
+
+To integrate React Native into your existing application, you'll need to set up a JavaScript development environment. This includes installing Node.js to run the Expo CLI and Yarn to manage the project's JavaScript dependencies.
+
+- [Node.js (LTS)](https://nodejs.org/en/): The runtime to execute JavaScript code and Expo CLI.
+- [Yarn](https://yarnpkg.com/): A package manager for installing and managing JavaScript dependencies.
+- <PlatformTag platform="ios" /> [CocoaPods](https://cocoapods.org/): one of the dependency
+  management system available for iOS. CocoaPods is a Ruby
+  [gem](https://en.wikipedia.org/wiki/RubyGems). You can install CocoaPods using the version of Ruby
+  that ships with the latest version of macOS.
+
+Learn more from the [Set up environment guide](/get-started/set-up-your-environment/).
+
+## Create an Expo project
+
+First, create an Expo project inside your existing native project's root directory.
+
+<Terminal cmd={['$ npx create-expo-app exp']} />
+
+This command creates a new directory named **exp** that contains your new Expo project. While you can name the project anything, this guide uses **exp** for consistency. The new project includes an example TypeScript application to help you get started.
```

Comment:
can we name this something else? i think using `exp` can be a bit confusing. even 'my-app' or 'my-project' might be a bit more clear. exp/expo/npx etc for people just coming to this all kind of blend together i imagine

Reply:
got it, we can rename it to `my-project`, which makes it really obvious that they should choose something else but might make things tricky because they can't name it `expo` or `react-native` because of the package names